web/server/web_client.c

```diff
@@ -143,7 +143,19 @@ void web_client_request_done(struct web_client *w) {
             debug(D_WEB_CLIENT, "%llu: Closing filecopy input file descriptor %d.", w->id, w->ifd);
```

Comment:
I'm always getting `SSL Handshake error (error:00000001:lib(0):func(0):reason(1))` now.

Reply:
Actually, I get the problem with ^SSL=force. When I don't set it, I get forbidden 3 times and then it works. Very strange...

![Screenshot from 2019-05-28 12-12-46](https://user-images.githubusercontent.com/43294513/58470542-2fb92c00-8142-11e9-9880-c09414cc3cdd.png)

Reply:
This error is not happening more with this last pull, now I have one that is error:00000006:lib(0):func(0):reason(6), according with the information I read this is related with certificate that is not trust, and this is my case, because I am using self signed certificate.

Reply:
I don't know what's happening, but it really doesn't work at all. I saw a redirect to https://127.0.1.1:19999 that didn't make sense and I keep getting forbidden, even from Firefox

Reply:
From thiago: 
I updated both with the newest pull, and before this I also finish the redirect using Header instead a HTML page, but I kept the HTML page that redirects to simplify the life of a users that want to use the API to request something, with the page in hand they do not need to see the headers.